nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/event/impl/PostCommitEventExecutor.java

```diff
@@ -223,9 +216,7 @@ public EventBundleRunner(List<EventListenerDescriptor> listeners, EventBundle bu
 
         @Override
         public Boolean call() {
-            if (log.isDebugEnabled()) {
-                log.debug("Events postcommit execution starting in thread: " + Thread.currentThread().getName());
-            }
+            log.debug("Events postcommit execution starting in thread: {}", Thread.currentThread().getName());
```

Comment:
Please use a lambda, here and in several other `log.debug` statements for this commit.